features/users.feature

```diff
@@ -29,9 +29,10 @@ Feature: User stats
   Scenario: Update user profile
     Given I am logged in as contestant user with attributes:
       | name | Valentin Mihov |
-    And there is a finished contest with attributes:
-      | name        | Fall contest |
-      | practicable | true         |
+    And there is a contest_in_group_one with attributes:
+      | name          | Fall contest |
+      | practicable   | true         |
     And I am on the homepage
+    And I follow "група 1"
```

Comment:
This is a magic string here, as it comes from the special contest trait. Probably a better approach will be to first create the group explicitly and then allow to create a contest that belongs to a group with some name. You don't need to fix this, as fixing it goes a bit too far for this exercise :)